terraform/module-lambda-roles/main.tf

```diff
@@ -48,7 +48,10 @@ data "aws_iam_policy_document" "payment_reconciler_execution" {
       "s3:GetObject",
       "logs:PutDestination",
       "logs:DescribeResourcePolicies",
-      "logs:DescribeDestinations"
+      "logs:DescribeDestinations",
+      "ec2:CreateNetworkInterface",
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:DeleteNetworkInterface"
```

Comment:
Can we confirm whether all of the listed actions do not support resource-level permissions and that it is strictly necessary to specify _all_ resources (`*`) for this reason? Ideally actions would be bound to a small number of resources in order to limit the policy to only those resources to which it should have access.

Reply:
For this release the changes here are acceptably—we should return to this later on and scope the actions to only the required resources.